docs/decisions/0000-use-markdown-any-decision-records.md

```diff
@@ -7,15 +7,15 @@ Which format and structure should these records follow?
 
 ## Considered Options
 
-* [MADR](https://adr.github.io/madr/) 3.0.0 – The Markdown Any Decision Records
+* [MADR](https://adr.github.io/madr/) 4.0.0 – The Markdown Any Decision Records
```

Comment:
Oh we use MADR here. That's where I went with the GG Report portal repo but Eric reminded me we use `adr-tools` in our other projects. 

It doesn't really matter right now, but let's plan on moving this over to `adr-tools` later.

See: https://github.com/SU-SWS/ood-givegab-portal/pull/111#discussion_r1767642933

Reply:
Oh interesting - sure let's change it later